react-components/src/architecture/concrete/pointsOfInterest/ads/PointsOfInterestAdsProvider.ts

```diff
@@ -42,20 +43,12 @@ export class PointsOfInterestAdsProvider implements PointsOfInterestProvider<Ext
   async upsertPointsOfInterest(
     pois: Array<{ id: ExternalId; properties: PointsOfInterestProperties }>
   ): Promise<Array<PointsOfInterestInstance<ExternalId>>> {
+    const upsertRequestData = createUpsertRequestFromPois(pois);
```

Comment:
It's not important for now but can we have a type for the: 
`{ id: ExternalId; properties: PointsOfInterestProperties }` ?

Reply:
I think we have one - I'll fix :sweat_smile: